repair/row_level.cc

```diff
@@ -676,6 +676,28 @@ void flush_rows(schema_ptr s, std::list<repair_row>& rows, lw_shared_ptr<repair_
     }
 }
 
+struct global_reader_permit_meta {
```

Comment:
Nitpick: It was hard for me to understand this name global_reader_permit_meta. And it took me a while that "global" refers to "all shards" and not any other possible sense of the word, so the "global permit" is basically a vector of separate permits on all the different shards - it's not a "vector of global permits" like I thought initially while I was reading this.
Maybe a comment explaining the purpose of the structure could have helped.

Reply:
Sure.